### ISSUE TITLE
no ref found from compile handlebars when compiling graphql schema

### ISSUE THREAD

**Issue by thepont** [anvilco/spectaql#54]
The `compile-handlebars:compile" (compile-handlebars)` step is returning multiple errors no ref found when compiling and several queries are missing from the documentation output.

```

Running "compile-handlebars:compile" (compile-handlebars) task
{"msg":"no ref found","name":"timestamp"}
{"msg":"no ref found","name":"count"}
{"msg":"no ref found","name":"target"}
{"msg":"no ref found","name":"verb"}
{"msg":"no ref found","name":"subject"}
{"msg":"no ref found","name":"metadata"}
{"msg":"no ref found","name":"subjectId"}
{"msg":"no ref found","name":"targetId"}
{"msg":"no ref found","name":"verbType"}
{"msg":"no ref found","name":"subjectType"}
{"msg":"no ref found","name":"targetType"}
```

This seems to happen when you have a query and a mutation with the same name.

see the following [schema](https://gist.github.com/thepont/b0a899a36c870ccc92adfcab56506488)

Looking further into this seems like a fundamental issue with the way this has evolved, The JSON schema representation does seem to be able to adequately represent and api that contains such things.

**Comment by newhouse** [comment]
@thepont a fix is coming...

**Comment by newhouse** [comment]
@thepont published in `0.4.1`!

**Comment by thepont** [comment]
Oh cool, thanks @newhouse
